### ISSUE TITLE
add 'minItems' argument to function-call-argument-newline

### ISSUE THREAD

**Issue by michaelshum321** [eslint/eslint#13475]
**What rule do you want to change?**
function-call-argument-newline

add minItems like what `function-paren-newline` has
**Does this change cause the rule to produce more or fewer warnings?**
Probably fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**

new option just like how function-paren-newline does it [here](https://eslint.org/docs/rules/function-paren-newline)

**Please provide some example code that this change will affect:**

```js
// "function-call-argument-newline: ['error', {minItems:3}]

// bad:
const uwu = console.log(
	'bad',
	1);

const bad2 = console.log(123, 456, 'long', 'waytoolong', 'boing');         

// good:
const owo = console.log('hello','goodbye')
const good1 = console.log(
	123, 
	456, 
	'long', 
	'waytoolong', 
	'boingo');   
```

**What does the rule currently do for this code?**

The rule is either 'all function calls have each argument on a new line' or 'no fn calls should have arguments on a new line'

**What will the rule do after it's changed?**

if minItems = 3, then function calls with more than 3 arguments will require to be on newlines instead of on one line
function calls with less than 3 arguments should stay on one line (this also has some room for configurability)

**Are you willing to submit a pull request to implement this change?**

Yes!

**Comment by yeonjuan** [comment]
Hi  :) @michaelshum321
The ESLint team has decided to freeze stylistic rules (please see [changes-to-rules-policies](https://eslint.org/blog/2020/05/changes-to-rules-policies) for more details).
So I suggest making an eslint-plugin or custom rule for your purpose.
IMO, it can be easily customized by using [eslint-rule-composer](https://github.com/not-an-aardvark/eslint-rule-composer).
